c_glib/test/test-month-interval-array.rb

```diff
@@ -0,0 +1,65 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+class TestMonthIntervalArray < Test::Unit::TestCase
+  include Helper::Buildable
+  include Helper::Omittable
+
+  def test_new
+    after_month = 1
```

Comment:
Did you copy `after_` prefix from other test such as `after_epoch` in `test-timestamp-array.rb`?
The `after_` prefix means "a value after epoch".
But `after_` prefix here is meaning because there is not a base value such as epoch.

How about simply using `raw_data = [0, 1]` here?